text/1006-simplified-cli.rst

```diff
@@ -97,19 +97,17 @@ the EdgeDB RC1 release::
     project list               List all projects
     project unlink             Clean-up the project configuration
 
-    server info                Show locally installed EdgeDB servers
-    server install             Install an EdgeDB server locally
-    server uninstall           Uninstall an EdgeDB server locally
-    server upgrade             Upgrade installations and instances
-    server list-versions       List available and installed versions of the server
-
     instance create            Initialize a new server instance
     instance status            Show statuses of all or of a matching instance
     instance start             Start an instance
     instance stop              Stop an instance
     instance restart           Restart an instance
-    instance destroy           Destroy an instance and remove the data
+    instance destroy           Destroy a local instance and remove the data
```

Comment:
Why emphasis on "local" here?

Reply:
Hm, now I'm not sure what concern I had in that editing instant. I'll drop "local".